presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -571,14 +571,14 @@ public void testMaxMinStringWithNulls()
     public void testGroupByPartitioningColumn()
             throws Exception
     {
-        assertQuery("SELECT orderkey, count(*) FROM lineitem GROUP BY orderkey");
+        assertQuery("SELECT orderkey, count(*) FROM lineitem WHERE orderkey % 8 = 0 GROUP BY orderkey");
     }
 
     @Test
     public void testJoinPartitionedTable()
             throws Exception
     {
-        assertQuery("SELECT * FROM lineitem a JOIN lineitem b ON a.orderkey = b.orderkey");
+        assertQuery("SELECT * FROM lineitem a JOIN lineitem b ON a.orderkey = b.orderkey WHERE a.orderkey % 8 = 0 ");
```

Comment:
can we join orders vs lineitem, instead?

@haozhun, what was the change you made trying to achieve?